test/unit/utils/telemetry/data_series_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'inspec/utils/telemetry'
+require 'json'
+require_relative '../../../helper.rb'
+
+class TestTelemetryDataSeries < Minitest::Test
+  def test_name
+    ds = Inspec::Telemetry::DataSeries.new('fizz')
+    refute_nil ds
+    assert_equal 'fizz', ds.name
+  end
+
+  def test_data
+    ds = Inspec::Telemetry::DataSeries.new('fizz')
+    refute_nil ds.data
+    assert_kind_of Array, ds.data
+    assert_empty ds.data
+  end
+
+  def test_data_append
+    ds = Inspec::Telemetry::DataSeries.new('fizz')
+    assert_empty ds.data
+    assert ds << 'foo'
+    assert_equal ['foo'], ds.data
+  end
+
+  def test_data_push_alias
+    ds = Inspec::Telemetry::DataSeries.new('fizz')
+    assert_empty ds.data
+    assert ds.push 'bar'
+    assert_equal ['bar'], ds.data
+  end
+
+  def test_to_h
+    ds = Inspec::Telemetry::DataSeries.new('fizz')
+    ds << 'foo'
+    assert_kind_of Hash, ds.to_h
```

Comment:
I think this should test the contents of the Hash.  Just from reading this, I'm not sure what the contents would be - what are the important keys?  What types are their values? Symbols or Strings for keys?

Reply:
Ok I can improve this.